src/test/java/com/hedvig/notificationService/service/FirebaseNotificationServiceImplTest.kt

```diff
@@ -98,6 +98,28 @@ internal class FirebaseNotificationServiceImplTest {
         deepMatchMessageIOSData(messages[0], title, body)
     }
 
+    @Test
+    fun sendNewMessageNotificationWithMessage() {
+        val title = "title"
+        val body = "body"
+        val message = "example message"
+        every { localizationService.getTranslation("DEFAULT_TITLE", any()) } returns title
+        every { localizationService.getTranslation("NEW_MESSAGE_BODY", any()) } returns body
+        classUnderTest.sendNewMessageNotification(MEMBER_ID, message)
+
+        deepMatchMessageCommonData(messages[0], mapOf("TYPE" to "NEW_MESSAGE"))
+        deepMatchMessageAndroidData(
+                messages[0],
+                mapOf(
+                        "TYPE" to "NEW_MESSAGE",
+                        "DATA_MESSAGE_TITLE" to title,
+                        "DATA_MESSAGE_BODY" to body,
+                        FirebaseNotificationServiceImpl.DATA_NEW_MESSAGE_BODY to message
```

Comment:
```suggestion
                        "DATA_NEW_MESSAGE_BODY" to message
```
I don't think you should use the constant in the test. Otherwise, someone could change the constants and the test  would still pass but the clients would not be happy about that :slightly_smiling_face:

Reply:
Very good point